data/json/recipes/recipe_food.json

```diff
@@ -4644,6 +4644,100 @@
   ]
 },{
   "type" : "recipe",
+  "result": "hickory_nut_unshelled",
+  "category": "CC_FOOD",
+  "subcategory": "CSC_FOOD_VEGGI",
+  "skill_used": "cooking",
```

Comment:
If it uses 0 cooking and 1 survival, it should probably not use cooking at all, just use the survival:1.